docs/build-mvm/how-to/schedule_functions.md

```diff
@@ -1 +1,883 @@
 # Cloudless Scheduler
+
+Tested on:
+
+- the FCLI version: `@fluencelabs/cli/0.13.5 linux-x64 node-v18.19.0`
+- Network env: `local`
+
+**TODO: when all updates are published, update to `kras` and the latest FCLI**
```

Comment:
Which major FCLI version will be used? On which to test?

Reply:
my guess is  >= 0.15.2; ideally testnet (dar) but not just yet.